chunkmaster/src/main/java/com/example/chunkmaster/HeartbeatController.java

```diff
@@ -0,0 +1,16 @@
+package com.example.chunkmaster;
+
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/chunkMaster")
+public class HeartbeatController {
+    @PostMapping("/chunkServerHeartBeat")
+    public void handleChunkServerHeartBeat(@RequestBody ChunkServerHeartbeat chunkServerHeartbeat){
```

Comment:
Is this receiving a heartbeat from ?

Reply:
this was added as an empty method, to test heartbeat endpoint, that request sent by chunkserver is received by chunkmaster.